docs/segment-management-on-carbondata.md

```diff
@@ -32,7 +32,7 @@ concept which helps to maintain consistency of data and easy transaction managem
 
   ```
   SHOW [HISTORY] SEGMENTS
-  [FOR TABLE | ON] [db_name.]table_name [LIMIT number_of_segments]
+  [FOR TABLE | ON] [db_name.]table_name [INCLUDE STAGE] [LIMIT number_of_segments]
```

Comment:
suggest to use "WITH STAGE"

Reply:
can user use "with stage" and segment query together?

Reply:
modified